docs/libcurl/opts/CURLMOPT_MAX_CONCURRENT_STREAMS.3

```diff
@@ -20,18 +20,18 @@
 .\" *
 .\" **************************************************************************
 .\"
-.TH CURLMOPT_MAX_HTTP2_CONCURRENT_STREAMS 3 "25 Apr 2019" "libcurl 7.64.0" "curl_multi_setopt options"
+.TH CURLMOPT_MAX_CONCURRENT_STREAMS 3 "14 May 2019" "libcurl 7.64.0" "curl_multi_setopt options"
 .SH NAME
-CURLMOPT_MAX_HTTP2_CONCURRENT_STREAMS \- set max concurrent streams for http2
+CURLMOPT_MAX_CONCURRENT_STREAMS \- set max concurrent streams for http2
 .SH SYNOPSIS
 #include <curl/curl.h>
 
-CURLMcode curl_multi_setopt(CURLM *handle, CURLMOPT_MAX_HTTP2_CONCURRENT_STREAMS, long max);
+CURLMcode curl_multi_setopt(CURLM *handle, CURLMOPT_MAX_CONCURRENT_STREAMS, long max);
 .SH DESCRIPTION
 Pass a long indicating the \fBmax\fP. The set number will be used as the
 maximum number of concurrent streams for a connections that libcurl should 
 support.
-
+Valid values range from 1 to 2147483647 (2^31 - 1) and defaults to 100.
 This option is for the multi handle's use only.
```

Comment:
I don't think this needs to say it is for multi handles only. The option is only for a single function and that function is for a multi handle.

I think the description needs to say something about this value being a request and that there's no guarantee that the set value will actually be used.

Reply:
> 
> 
> I don't think this needs to say it is for multi handles only. The option is only for a single function and that function is for a multi handle.
> 
Agree
> I think the description needs to say something about this value being a request and that there's no guarantee that the set value will actually be used.
Actually upstream nghttp2 doesn't mention something like this. May be I can only update that its the max value and would be dependent on transfer traffic rate.